articles/virtual-machines/generalize.yml

```diff
@@ -106,7 +106,8 @@ procedureSection:
       - |   
         Verify if CD/DVD-ROM is enabled. If it is disabled, the Windows VM will be stuck at out-of-box experience (OOBE).  
         ```
-          Registry key Computer\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\cdrom\start (Value 4 = disabled, expected value 1 = automatic) Make sure it is set to 1.
+        REM Enable CD/DVD-ROM
```

Comment:
Can you make sure like 109 and 110 are written in proper sentences? It doesn't look like code to me, so please use full sentences with punctuation, etc. Ping me on Teams with this PR link when you are ready and I can sign off.